src/pages/Search/SearchBooks.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import '../../styles/theme';
+import styled from 'styled-components';
+
+const filters = [
+  { filter: '전체', placeholder: '검색어를 입력하세요' },
+  { filter: '저자', placeholder: '저자를 입력하세요' },
+  { filter: '제목', placeholder: '제목을 입력하세요' },
+  { filter: '출판사', placeholder: '출판사를 입력하세요' },
+];
+
+export default function SearchBooks() {
+  const [filter, setFilter] = useState('전체');
+  const [visible, setVisible] = useState(false);
+  const [placeholder, setPlaceholder] = useState('검색어를 입력하세요.');
+  const [searchValue, setSerchValue] = useState('');
+
+  const handleFilter = (filter, placeholder) => {
+    setFilter(filter);
+    setPlaceholder(placeholder);
+    setVisible(false);
+  };
+
+  const onChangeSearchValue = e => {
+    setSerchValue(e.target.value);
+  };
+
+  const handleCleanSearch = e => {
+    setSerchValue('');
+  };
+
+  return (
+    <div>
+      <Filtering>
+        <FilterBtn
+          onClick={() => {
+            setVisible(!visible);
+          }}
+        >
+          {filter}
+          <IconBox>
+            <i className="fas fa-angle-down" />
+          </IconBox>
+        </FilterBtn>
+        {visible && (
+          <FilterWrapper>
+            {filters.map(({ filter, placeholder }) => {
+              return (
+                <Filters
```

Comment:
```suggestion
                <Filter
```
여기도 위에 말씀드렸던 내용과 동일한 이유로 단수로 써주는게 좋겠습니다

Reply:
수정 완료했습니다